### PR TITLE
update inbox section

### LINKED ISSUE #56: Current recommendation for public inboxes can lead to spam
the issue for tackling the use of public inboxes more broadly:  https://github.com/solid/specification/issues/464

https://solid.github.io/webid-profile/#inbox

> If no inbox is found a Pod Management App MAY create an inbox by creating a container. In that case, the app SHOULD also create access controls for the container that give read and write permissions to the WebID owner and append but not read or write permissions to everyone else.

I see this as a very risky suggestion. If the pod management app actually does, it can make storage vulnerable to spam. I think we should handle this issue on the spec level since in SAI we also make very minimal use of a specialized public inbox and it will require special considerations to prevent spam.

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
This PR updates the Inbox section with changes based on discussions. 

Closes #56

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1389,27 +1389,14 @@ <h2 property="schema:name">Storage</h2>
             <h2 property="schema:name">Inbox</h2>
             <div datatype="rdf:HTML" property="schema:description">
               <p>
-                A Solid inbox is a Solid-specific messaging system that is
-                similar to but not the same as an email inbox.
+                A Solid WebID Profile can advertise an inbox with the <code>ldp:inbox</code> property to allow applications to send
+                and discover
+                notifications for the attention of the owner of the WebID.
```
Comment (csarven) [inline index.html:1394]:
```suggestion
                A Solid WebID Profile can advertise an inbox with the <code>ldp:inbox</code> property [<cite><a class="bibref" href="#bib-ldn">LDN</a></cite>] to allow applications to send
                and discover notifications about activities, interactions, and new information
                for the attention of the owner of the WebID.
```

**Review by csarven — APPROVED** [review]

**Review by jeff-zucker — CHANGES_REQUESTED** [review]
I find this to be too brief and unhelpful.  I agree with removing the conformance wording as discussed previously.  I do not recall saying that we would remove a recommendation for creating a missing inbox. I believe we need to distinguish this from a regular email box.  I think we also need to make a strong recommendation that the predicate be used and recommend that it be placed where it can most easily be found.  

Also, my recollection is that we decided that the inbox would have a cardinality of one.   I'll leave that out for now but we should discuss.  If the cardinality can be greater than one, we should recommend how an app should deal with the situation.

Is there an appropriate link in LDN we might add?

I propose this instead :  

<p>
A Solid inbox is a Solid-specific messaging system that is similar to but not the same as an email inbox, see <a href="https://solidproject.org/TR/protocol#resource-containment">resource
containment</a>. A Solid Profile can advertise an inbox with a triple in the Solid Profile Document in this form :  <code><?WebID> ldp:inbox &lt;?IRI-of-a-container></code.> to allow applications to send and discover notifications for the attention of the owner of the WebID.
</p>

<p>
As the Solid inbox is an important feature of a profile,  when an app capable of repairing a profile encounters a Solid Profile without an `ldp:inbox` triple, it can prompt the user to create one.   It is recommended that apps place the inbox triple in the Solid Profile Document, making it easy to  find  early in the discovery process.  
</p>

**Comment by csarven** [comment]
> If the cardinality can be greater than one

Client behaviour for more than one is unspecified with the understanding that only one inbox needs to be discovered towards interop ( https://www.w3.org/TR/ldn/#discovery ).

I prefer the lean version in the current PR, simply the discovery of a WebID's inbox (and notifications then after) from a WebID Profile Document, and assigning an inbox.

Worthwhile to note that an inbox can also be associated to a resource with the Link header. However, I think that interaction is out of the scope for this spec.

**Comment by VirginiaBalseiro** [comment]
Responding to @jeff-zucker's comments: 

> ...in this form :

> <?WebID> ldp:inbox <?IRI-of-a-container>

Covered by link to LDN, where there are examples (see example 6 in Discovery section).

------- 

> Also, my recollection is that we decided that the inbox would have a cardinality of one. I'll leave that out for now but we should discuss. If the cardinality can be greater than one, we should recommend how an app should deal with the situation.

Covered by link to LDN, which states: 

> A resource must advertise only one Inbox. One Inbox may be used by multiple resources, for example using the same Inbox for replies to all of my blog posts and shares of all of my photos.

-------

> A Solid inbox is a Solid-specific messaging system that is similar to but not the same as an email inbox. 

I find this to be potentially confusing to the reader. This is covered by: 

> allow applications to send and discover notifications about activities, interactions, and new information for the attention of the owner of the WebID.

--------

> As the Solid inbox is an important feature of a profile, when an app capable of repairing a profile encounters a Solid Profile without an `ldp:inbox` triple, it can prompt the user to create one. It is recommended that apps place the inbox triple in the Solid Profile Document, making it easy to find early in the discovery process.

This is covered by: 

> Applications can create an inbox (see <a href="https://solidproject.org/TR/protocol#resource-containment">resource containment</a>) as needed with desired permissions and policies, and then associate the inbox to a WebID by updating the WebID Profile Document.

**Comment by jeff-zucker** [comment]
> Covered by link to LDN

Sorry,  I wrote my response before Sarven's change and I did not see any link to LDN.  I withdraw all of my suggested changes except for these two points

1) I think we ought to inform developers about the cardinality for purposes of discoverability, something along the lines of : As mentioned in the LDN specification, "a resource may  advertise only one inbox".  The behavior for more than one inbox is not specified.

2) The LDN spec does not say anything about where triples pointing to the inbox should go, therefore I believe this sentence should be retained  :

> >  It is recommended that apps place the inbox triple in the Solid Profile Document, making it easy to find early in the discovery process.

**Review by jeff-zucker — APPROVED** [review]

**Comment by VirginiaBalseiro** [comment]
Added sentence based on @jeff-zucker's comments on https://github.com/solid/webid-profile/pull/69/commits/71f0751a1647cbcbc31aa50c70c04a6e0bab72a8

**Pull request merged**